UI/window-basic-main.cpp

```diff
@@ -3805,6 +3809,53 @@ QMenu *OBSBasic::AddScaleFilteringMenu(obs_sceneitem_t *item)
 	return menu;
 }
 
+QMenu *OBSBasic::AddBackgroundColorMenu(obs_sceneitem_t *item)
+{
+	QMenu *menu = new QMenu(QTStr("ChangeBG"));
+	QAction *action;
+
+	obs_data_t *privData = obs_sceneitem_get_private_settings(item);
+	obs_data_release(privData);
+
+	obs_data_set_default_int(privData, "color-preset", 0);
+	int preset = obs_data_get_int(privData, "color-preset");
+
+	action = menu->addAction("Clear", this, \
+		+ SLOT(ColorChange()));
+	action->setCheckable(true);
+	action->setProperty("preset", 0);
+	action->setChecked(preset == 0);
+
+	action = menu->addAction("Custom Color", this, \
+		+ SLOT(ColorChange()));
+	action->setCheckable(true);
+	action->setProperty("preset", 1);
+	action->setChecked(preset == 1);
+
+	menu->addSeparator();
+
+	QWidgetAction *widgetAction = new QWidgetAction(menu);
+	ColorSelect *select = new ColorSelect(menu);
+	widgetAction->setDefaultWidget(select);
+
+	for (int i = 1; i < 9; i++) {
+		stringstream button;
+		button << "preset" << i;
+		QPushButton *colorButton = select->findChild<QPushButton *>(
+				button.str().c_str());
+		if (preset == i + 1) {
```

Comment:
Style:  This `if` doesn't need curly braces, so you can remove them, but leave a blank line after the `if` contents.